.env.example

```diff
@@ -0,0 +1,77 @@
+# ==============================================================================
+# 0. METABASE DEVELOPMENT ENVIRONMENT
+# ==============================================================================
+#
+# Central configuration for local Metabase development.
+# Copy to .env and uncomment/modify values as needed.
+#
+# Usage:
+#   cp .env.example .env
+#
+# Hint: Auto-load environment variables with direnv or mise:
+#   direnv  - Create .envrc with: dotenv .env       (https://direnv.net)
+#   mise    - mise.toml: [env] _.file = ".env"      (https://mise.jdx.dev)
+#
+# Full Metabase environment configuration reference:
+# https://www.metabase.com/docs/latest/configuring-metabase/environment-variables
+
+
+# ==============================================================================
+# 1. E2E / CYPRESS TESTING
+# ==============================================================================
+#
+# Configuration for running Cypress end-to-end tests locally.
+#
+# The whole environment is already orchestrated via deps.edn using :e2e alias.
+# Treat it as a source of truth for JVM flags and ENVs whether you decide to run
+# backend on your own, or using `yarn test-cypress` (local test runner).
+#
+# You don't _have to_ export any of these vars (except for the tokens below).
+# Things will still work.
+
+# 1.1. E2E Application Database (H2 required)
+# ------------------------------------------------------------------------------
+
+# Whether to run the enterprise or the open-source Metabase version
+MB_EDITION="ee"
+
+# Which file to use for the ephemeral testing application database?
+MB_DB_FILE="./e2e/tmp/cypress-test.db"
+
+# Where does Cypress go to look for the application?
+MB_JETTY_PORT=4000
+
+# 1.2 E2E Local Runner Setup
+# ------------------------------------------------------------------------------
+
+# Whether to run true e2e test suite, or SDK component tests (e2e | component)
+CYPRESS_TESTING_TYPE="e2e"
+
+# Whether to start Cypress in an interactive (open) mode with full GUI
+CYPRESS_GUI="true"
+
+# Whether to generate application database dumps (colloquially "snapshots")
+# Unless you're experimenting, you generally want to leave this on.
+GENERATE_SNAPSHOTS="true"
+
+# 1.3 E2E Tokens
+# ------------------------------------------------------------------------------
+#
+# If you're Metabase core dev, get them here:
+# https://www.notion.so/metabase/2b769354c9018070bb65f4e6a1a399fe#2b769354c90180479352da1047fda824
+
+CYPRESS_MB_ALL_FEATURES_TOKEN="redacted"
+CYPRESS_MB_STARTER_CLOUD_TOKEN="redacted"
+CYPRESS_MB_PRO_CLOUD_TOKEN="redacted"
+CYPRESS_MB_PRO_SELF_HOSTED_TOKEN="redacted"
```

Comment:
Have you seen this line?
https://github.com/metabase/metabase/pull/66431/files#diff-a3046da0d15a27e89f2afe639b25748a7ad4d9290af3e7b1b6c1a5533c8f0a8cR61